anomalous-value-js/src/agent.js

```diff
@@ -6,8 +6,9 @@ const { Finding, FindingSeverity, FindingType, getJsonRpcUrl } = require("forta-
 const contractAddresses = {};
 const provider = new ethers.providers.getDefaultProvider(getJsonRpcUrl());
 
-function provideHandleTransaction(provider) {
+function provideHandleTransaction(contractAddresses, provider) {
   return async function handleTransaction(txEvent) {
```

Comment:
I think a description of the handler would be useful here--you have to read into the code to figure out that this agent records transaction values for all contracts that it processes and identifies unusually high transaction values.

Reply:
Included the following comment when `contractAddresses` is initialized: "Need to initialize a dictionary whose keys will be contract addresses the agent encounters and whose values are RollingMath objects tracking the ether values of transactions that touch the contract"